pulpcore/app/models/task.py

```diff
@@ -287,11 +287,11 @@ def current():
             pulpcore.app.models.Task: The current task.
         """
         try:
-            task_id = get_current_job().id
+            task_job_id = get_current_job().id
         except AttributeError:
             task = None
         else:
-            task = Task.objects.get(pk=task_id)
+            task = Task.objects.get(job_id=task_job_id)
```

Comment:
```job_id``` is a UUID, ```id``` is an integer AutoField, so wouldn't this + the next line be invalid?